README.md

```diff
@@ -27,18 +27,53 @@ Another moving part is the new-ish Annotations 2.0 which uses Leaflet.Illustrate
 
 ## Quick install
 
-We provide an install script for Amazon's Cloud9 service, which provides standard Linux virtual machines with a web-based editor: https://aws.amazon.com/cloud9/
+We provide an install script for Amazon's Cloud9 service, which provides standard Linux virtual machines with a web-based editor: https://c9.io/login
+
 
 To use it:
 
+### Installation (do this only once):
+
 1. Click `Create new workspace`
+
 2. Name your workspace and enter `https://github.com/USERNAME/mapknitter` (where `USERNAME` is your GitHub username) under `Clone from Git or Mercurial URL`
 3. Choose `Ruby` template
+
 4. Click `Create workspace`
-5. Once it loads, in the `bash` console at the bottom of the screen, type `source install_cloud.sh`
-6. Press `Control-x` when prompted for a message.
-7. Wait for the installation to run, and when it's complete, press the `Run` button at the top of the page.
-8. Open the URL which pops up to see MapKnitter booted up. Great work!
+
+5. Once it loads in the `bash` console at the bottom of the screen, pull in the latest `main` branch (note: AWS only lets us track `master` by default. Since we're deprecating in a month, quick workaround):
+
+```Bash
+$ git fetch --all
+
+$ git checkout remotes/origin/main
+
+$ git checkout -b main
+
+$ git remote add upstream https://github.com/publiclab/mapknitter.git 
```

Comment:
shouldn't we `git remote update` after adding a new remote? :thinking:

Reply:
Agree. Try adding new remote first, then fetch them all, and then reset your origin's main with upstream (`git reset --hard upstream/main`) so as to avoid merge conflicts.

Reply:
@gauravano @alaxalves can one of you please try making a commit suggestion because I'm not sure exactly what changes you want! 

also for the switch hoping we don't have to make the solution like this for codenvy and theres a built-in way to do it there - but the merge conflict will appear there too just as it does here so for now this would have to be the solution there too and we can work on getting rid of it